.huskyrc

```diff
@@ -1,6 +1,6 @@
 {
-    "hooks": {
-        "pre-commit": "lint-staged",
-        "pre-push": "npm run typecheck && npm run lint && npm run stylelint"
```

Comment:
There is no styelint command in the package.json, and there's no need to run the linter both before commit and push (right?).